app/test_split_integer.py

```diff
@@ -1,21 +1,21 @@
 from app.split_integer import split_integer
 
 
-def test_sum_of_the_parts_should_be_equal_to_value():
-    pass
+def test_sum_of_the_parts_are_equal_to_value() -> None:
+    assert split_integer(17, 4) == [4, 4, 4, 5]
```

Comment:
in this test you need to get sum or the function result and check that it is equal to the initial value